### PR TITLE
Add Pinboard import

### PR DESCRIPTION
**Pull request by j0k3r**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | yes
| Translation   | yes
| Fixed tickets | fix https://github.com/wallabag/wallabag/issues/2480
| License       | MIT

Add Pinboard import (from the json export)

- [x] Missing doc

### CODE REVIEW

File: tests/Wallabag/ImportBundle/fixtures/pinboard_export [hunk 1]
```diff
@@ -0,0 +1,5 @@
...
+{"href":"https:\/\/ilia.ws\/files\/nginx_torontophpug.pdf","description":"Nginx Tricks for PHP Developers","extended":"","meta":"9adbb5c4ca6760e335b920800d88c70a","hash":"0189bb08f8bd0122c6544bed4624c546","time":"2016-10-05T07:11:27Z","shared":"yes","toread":"no","tags":"nginx PHP best_practice"},
+{"href":"https:\/\/jolicode.com\/blog\/starting-a-mobile-application-with-react-native","description":"Starting a mobile application with React Native","extended":"While preparing our next React Native training, I learnt a lot on the library and discovered an amazing community with a lot of packages.","meta":"bd140bd3e53e3a0b4cb08cdaf64bcbfc","hash":"015fa10cd97f56186420555e52cfab62","time":"2016-09-23T10:58:20Z","shared":"yes","toread":"no","tags":"react-native"},
```
Comment (nicosomb) [inline tests/Wallabag/ImportBundle/fixtures/pinboard_export]: Hello @JoliCode team 👋 !

**Comment by j0k3r** [comment]
We might need german translation for the doc part (it seems to miss some older part too, like Readability, etc.)

**Comment by Strubbl** [comment]
@j0k3r can you please add the docs part, which is missing, as English text to the German docs?

**Comment by j0k3r** [comment]
@Strubbl added

**Comment by j0k3r** [comment]
Build fixed by https://github.com/hoaproject/Stream/pull/20

**Pull request merged**